src/db/tables/onerep_scans.ts

```diff
@@ -168,6 +168,18 @@ async function getScansCount(
     .andWhere("onerep_scan_reason", scanReason);
 }
 
+async function getScansCountForProfile(
+  onerepProfileId: number
+): Promise<number> {
+  return parseInt(
```

Comment:
Why are we casting as string and then calling `parseInt()` to convert back to a number?
Wouldn't knex's `.count()` be returning a number directly, or is the count value so high it's converting to a string in some cases here?

Reply:
The .count() can return number or string (empty) I believe. So we cannot just infer `as number`